enforcer-rules/src/main/java/org/apache/maven/plugins/enforcer/BanDynamicVersions.java

```diff
@@ -110,7 +117,8 @@
 
     /**
      * Specify the ignored dependencies. This can be a list of artifacts in the format
-     * <code>groupId[:artifactId][:version]</code>. Any of the sections can be a wildcard by using '*' (ie group:*:1.0)
+     * <code>groupId[:artifactId[:version[:type[:scope:[classifier]]]]]</code>. 
+     * Any of the sections can be a wildcard by using '*' (ie group:*:1.0).
```

Comment:
`.e.g {@code group:*:1.0}`

Reply:
fixed in https://github.com/apache/maven-enforcer/pull/187/commits/2d82f485ff5c73a6052b189fa9488d02fa4b3cbb.